catalog/transformer/catalog-transformer-pptx/src/test/java/ddf/catalog/transformer/input/pptx/TestPptxInputTransformer.java

```diff
@@ -36,19 +36,19 @@
 import ddf.catalog.transform.InputTransformer;
 import ddf.catalog.transformer.input.tika.TikaInputTransformer;
 
-public class PptxInputTransformerTest {
+public class TestPptxInputTransformer {
```

Comment:
Looks like we should be using PptxInputTransformerTest instead per the style guide.